pkg/fakes/registy.go

```diff
@@ -64,6 +66,11 @@ type Registry struct {
 	invocationsMutex sync.RWMutex
 }
 
+// TODO(khouzam)
+func (fake *Registry) DownloadImage(arg1 string, arg2 string) error {
```

Comment:
nit: you should be able to generate the fakes using `make generate-fakes`

Reply:
I didn't know that.  Thanks @prkalle!  I will update.